src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -0,0 +1,36 @@
+package codeu.controller;
+
+        import org.junit.Before;
+        import org.junit.Test;
+        import org.mockito.Mockito;
+
+        import javax.servlet.RequestDispatcher;
+        import javax.servlet.ServletException;
+        import javax.servlet.ServletOutputStream;
+        import javax.servlet.http.HttpServletRequest;
+        import javax.servlet.http.HttpServletResponse;
+        import java.io.IOException;
+
+public class ProfileServletTest {
+    private ProfileServlet profileServlet;
+    private HttpServletRequest mockRequest;
+    private HttpServletResponse mockResponse;
+    private ServletOutputStream mockOutputStream;
+    private RequestDispatcher mockRequestDispatcher;
+
+    @Before
+    public void setup() throws IOException {
```

Comment:
What does this specifically do?

Reply:
I am not being able to pass all unit test. I used this from ConversationServlettest. I figured out many variables here are useless to run Prototypes.